src/nupic/research/TP_shim.py

```diff
@@ -105,6 +105,21 @@ def compute(self, bottomUpInput, enableLearn, computeInfOutput=None):
     return output
 
 
+  def topDownCompute(self, topDownIn=None):
```

Comment:
I tried but I couldn't get the inheritance working correctly when I did. I can try again, but we should probably work to switch to the new `TemporalMemoryRegion` that Jared created instead.